packages/js/src/utils.ts

```diff
@@ -91,3 +94,29 @@ export function hashContract(contract: string | Buffer): string {
 }
 
 export const EMPTY_CONTRACT_HASH = '11111111111111111111111111111111';
+
+/**
+ *
+ * @returns network to connect to. Default 'sandbox'
+ */
+export function getNetworkFromEnv(): 'sandbox' | 'testnet' {
```

Comment:
Should we add `mainnet` now?

Reply:
Sure!

Reply:
In retrospect I think it shouldn't be added to this PR as it involves more internal efforts.